### DISCUSSION TITLE
Save position of Volume Mixer window

### DISCUSSION THREAD

**Discussion by emperortomato** [File-New-Project/EarTrumpet#687] [Ideas]
It would be cool if EarTrumpet could save the position of the Volume Mixer window after it has been moved, and then open in that same position next time. At the moment, no matter where I move it, if I close & re-open it, it opens in the upper left corner of my screen.

This is somewhat related to [349](https://github.com/File-New-Project/EarTrumpet/issues/349) (which I understand is a Windows bug), in the sense that moving the mixer is a difficult and time-consuming ordeal at the moment.

**Comment by davuxcom** [comment]
I looked into this and it's a little more complicated than one may expect.  I agree it's made much worse due to #349
